concepticondata/conceptlists/Wang-2021-90.tsv

```diff
@@ -0,0 +1,91 @@
+Wang-2021-90-82	82	1495	HAPPY	comfortable	word82	舒心	a	2	4	0.0334804302	0.006236274634	0.01774503191	0.003097627411	0.03352057866	0.006253782234	0.03774770369	0.008893270778	0.2729474838	0.05289269009	0.06269500217	-0.8076586059	3.961538462	4.222222222	1.178571429	1.44	1.481481481	3.041666667	5.481481481	1.482720991	1.867399124	0.7723735149	0.916515139	0.9754829325	1.781039429	0.7529618629	-1.01523	-1.10004	0.59998	0.31055	16	6.038461538	0.903089987	2.281052012	3.319993768	2.48641475	4.000051897	-0.91877	-0.34044	32.12494475	15.70832091	44.14936326
+Wang-2021-90-83	83	1495	HAPPY	happy	word83	快乐	a	2	4	0.04423149278	0.009923482492	0.0200259638	0.004105700491	0.04415250202	0.008541540547	0.0584581563	0.01191872065	0.2728484513	0.05066767405	0.06549503615	-0.2757063514	4.730769231	4.740740741	1.107142857	1.4	2.037037037	4.583333333	5.888888889	1.563526586	1.788695111	0.4162696521	1.224744871	1.786304339	1.742478661	0.9336995618	-0.70073	-0.3905	0.71144	1.07012	12	6.730769231	1.462397998	3.242911092	3.256819514	3.974467796	3.988891428	0.03511	0.64321	22.50593047	13.06911048	36.30115198
+Wang-2021-90-84	84	174	PROUD	proud	word84	骄傲	a	2	4	0.03145450163	0.005865044894	0.0151245445	0.002952928042	0.03366341507	0.006280536607	0.03313000151	0.00697145305	0.2593939418	0.05950284234	0.06085517634	-0.9604875299	4.192307692	3.592592593	1.035714286	1.36	2.037037037	4.25	4.666666667	1.673779693	1.692955069	0.1889822365	0.9073771726	1.786304339	1.939296152	1.414213562	-1.07322	-0.11805	0.81577	1.9878	21	6.461538462	1.602059991	1.663988162	1.734083813	3.271466418	4.426572384	-0.72245	0.11348	24.45721051	9.827849532	36.94510541
+Wang-2021-90-85	85	82	ANGER	anger	word85	愤怒	a	2	4	0.04005928864	0.006585400663	0.01821089358	0.003309335826	0.04081678983	0.006189052727	0.0343721893	0.009600046808	0.3659423123	0.03670863539	0.07763202763	-0.1384034914	4.846153846	4.962962963	1.071428571	1.2	3.740740741	4.541666667	2.37037037	1.376729681	1.764641639	0.2622652642	0.5	1.933362806	1.768791449	0.926039879	-0.27404	1.75633	0.98938	1.07741	21	6.653846154	1.113943352	1.735388088	1.793620683	3.974467796	4.163235164	0.30061	0.71857	30.53492014	17.01585304	43.37297269
+Wang-2021-90-86	86	1419	FEAR (BE AFRAID)	fear	word86	恐惧	a	2	4	0.03011365818	0.006167222433	0.01360963905	0.003280461521	0.03145485275	0.005982231698	0.03046899692	0.008106471953	0.3549782846	0.04983125537	0.06929988303	-0.4385032139	4.346153846	4.740740741	1.071428571	1.36	4.666666667	5	2.074074074	1.598557042	1.872731296	0.2622652642	1.075484387	1.839732422	1.474419562	0.8738036184	-0.362	2.38965	1.17597	0.43951	21	6.269230769	1.176091259	2.188906558	1.990397467	3.974467796	3.974467796	0.12202	0.84633	21.72480537	9.301431693	32.26258148
```

Comment:
This is a noun. It should be mapped to a noun reading, not to the verbal reading. It is something like "scare" (Schrecken in German).

Reply:
I would in these cases check where we have mapped it before. Doing so (pasting the Chinese expression to https://concepticon.clld.org/values) shows that it is a unique word we have not encountered so far. It is more like a term you use to describe what one feels in a horror movie, so I'd say: we rather do not map it now.